### PR TITLE
move the handling of the TLM member.created when user is creating a workspace for his own

### PR DESCRIPTION
**Pull request by Skylsmoi**
https://github.com/tracim/tracim/issues/3230

## Checkpoints

The goal of this section is to help code reviewers and the merge request author to do the required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present
- [x] Automated tests have been written (covering the feature or non regression), or *at least* an issue has been created to implement the test

**For developers**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] The original issue has been updated with a short summary of the implemented solution

**For the quality team**

- [x] Tested by the quality team

### CODE REVIEW

File: frontend/src/container/ReduxTlmDispatcher.jsx [hunk 1]
```diff
@@ -75,16 +74,11 @@ export class ReduxTlmDispatcher extends React.Component {
     this.props.dispatch(removeWorkspace(data.workspace))
   }
 
-  handleUserModified = data => {
-    this.props.dispatch(updateUser(data.user))
-  }
```
Comment (gbonaspetti) [inline frontend/src/container/ReduxTlmDispatcher.jsx:80]: why remove that handler?

Comment (Skylsmoi) [reply frontend/src/container/ReduxTlmDispatcher.jsx:80]: it was duplicated in the file, the result of merge I guess.
Both had the exact same implementation

File: frontend/src/container/Sidebar.jsx [hunk 2]
```diff
@@ -42,9 +49,28 @@ export class Sidebar extends React.Component {
...
+    const tlmWorkspace = tlmFieldObject.workspace
+    const loggedUserId = props.user.userId
+
+    if (loggedUserId === tlmUser.user_id) {
+      props.dispatch(addWorkspaceList([tlmWorkspace]))
+      props.dispatch(addWorkspaceMember(tlmUser, tlmWorkspace.workspace_id, tlmFieldObject.member))
+
+      // INFO - CH - 2020-06-25 - if logged used is author of the TLM and the new role is for him, it means the logged
+      // user created a new workspace
+      if (loggedUserId === tlmAuthor.user_id) {
```
Comment (metzgegu) [inline frontend/src/container/Sidebar.jsx:66]: This mean that when a workspace is added by a user, each window where the user is connected to Tracim will be redirected to the dashboard of the new workspace ?

Comment (Skylsmoi) [reply frontend/src/container/Sidebar.jsx:66]: indeed, it's an issue.
I'm going to implement the identification through the client_token

**Pull request merged**